docs/containers/bridge-to-kubernetes.md

```diff
@@ -178,7 +182,7 @@ Additionally, you can find the diagnostic logs in the `Bridge to Kubernetes` dir
 
 ## Running in isolation mode
 
-With Bridge to Kubernetes, you can also set up an isolated version the services you're working on, meaning that others who are using the cluster won't be affected by your changes. This isolation mode is accomplished by routing your requests to your copy of each affected service, but routing all other traffic normally. More explanation on how this is done can be found at [How Bridge to Kubernetes Works][btk-overview-routing].
+With Bridge to Kubernetes, you can also set up an isolated version the services you're working on, meaning that others who are using the cluster won't be affected by your changes. This isolation mode is accomplished by routing your requests to your copy of each affected service, but routing all other traffic normally. More explanation on how this is done can be found at [How Bridge to Kubernetes Works](https://docs.microsoft.com/en-us/visualstudio/containers/overview-bridge-to-kubernetes?view=vs-2019#using-routing-capabilities-for-developing-in-isolation).
```

Comment:
This link has been fixed and will be published shortly. The issue was that square brackets were not used. See https://github.com/microsoft/vscode-docs/commit/3d83642e1f55a03863e74b05ad3df08c71af30c4